README.md

```diff
@@ -121,3 +123,24 @@ Note that, due to the way static files are managed for the normal development en
 | `batch_process`        | Given a list id argument run parse, geocode, and match via the batch_process Django management command |
 | `devhealthcheck.sh`    | Simulate application load balancer health checks in development                                        |
 | `postfacilitiescsv.py` | POST the rows of a CSV containing facility information to the facilities API                           |
+
+## Management
+
+### Making Superusers
+
+In staging and production we do not have access to the Django Shell so granting
+superuser permissions to a member of the OAR team or development team that needs
+to manage the system requires the use SQL statements to adjust the permissions.
+
+- Connect to the staging or production database via the bastion instance
```

Comment:
It seems like I don't have the bastion instance set up yet. Are there any documentation about setting it up?

Reply:
The bastions (one each for staging and production) are AWS EC2 instances. The details for connecting to them are not something I wanted to add to the public README. Feel free to schedule a time to pair and get this connectivity set up.